modules/transport/base/src/main/java/org/apache/axis2/transport/base/BaseUtils.java

```diff
@@ -191,17 +191,15 @@ public static boolean isBlank(String str) {
     }
 
     public static boolean isUsingTransport(AxisService service, String transportName) {
-        boolean process = service.isEnableAllTransports();
```

Comment:
Shouldn't we allow all transports to work when `service.isEnableAllTransports()` is set to true?

Reply:
The problem is, service. enableAllTransports is true by defult, where it should be false. We do not explicitely set it to true anywhere else. I did not change the default value since it is being used in many other places.

Reply:
This is bit smelly. Can we try to understand the purpose of enableAllTransports
and remove it if not required?